database/src/main/resources/application.properties

```diff
@@ -6,3 +6,13 @@ app.version=1.0.0
 database.storage.path=
 # Limit parameter value
 limit=100
+# Application port
+port=3306
```

Comment:
Please don't use the same port as for database, only one application can be run on one port. I think `8080` would be OK as default port.